### DISCUSSION TITLE
2013/11/21/existing-airway-beacons

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#9] [Announcements]
# 2013/11/21/existing-airway-beacons

http://localhost:4000/2013/11/21/existing-airway-beacons

**Comment by jengalas** [comment]
Posted by Alex
September 20, 2021 at 6:26 pm
_Imported from WordPress_

Do you have more information on the Kingsport beacon?

**Reply by jengalas** [reply]
Posted by Sean Plankey
March 27, 2023 at 10:14 am
_Imported from WordPress_

The old airway beacon is still mounted at the highest part of Chimney Top Mountain (3,118′ MSL) and is approximately 14.8nm WSW of the TRI airport. You can see its location on old (1930’s-1950’s) USGS topographic charts for Lovelace, TN. Using modern technology, you can find the beacon (and the old stubby little fire tower on the SW peak) by using Google Earth. Takes a little bit of patience but using my data, you’ll find it.

I hiked up to the fire tower and nav beacon yesterday (3/26/23) and I must say it was grueling. A combination of rocky terrain and punishing grades would kick the heck out of a mountain goat! The final ascent up the “chimney” is challenging and dangerous without ropes. You really gotta hug them rocks!

**Comment by jengalas** [comment]
Posted by michael
April 5, 2015 at 2:18 pm
_Imported from WordPress_

I found a dietic triangulation marker in nc. What are they.

**Reply by jengalas** [reply]
Hello Michael,

Thanks for your comment!

I’m assuming that you’re referring to a Geodetic Triangulation Station marker? These are one of many different types of survey markers in use for various purposes. Geodesy is the science of measuring and modeling the shape and other attributes (such as gravity, magnetism, and tides) of the earth. Both geodetic and land surveyors (the type you would hire to determine your property boundaries) use survey markers in their work, as do mapmakers and civil engineers—basically anyone who has a need to know their exact location and/or elevation.

A geodetic triangulation station marks a precise location on the earth. It is typically part of a network of many such marks called, in the U.S., the National Spatial Reference System. (For more, see: [http://www.ngs.noaa.gov/INFO/OnePagers/NSRSOnePager.pdf](http://www.ngs.noaa.gov/INFO/OnePagers/NSRSOnePager.pdf)) Triangulation stations are considered “horizontal control”: they mark a precise latitude/longitude position, such as you would see on your GPS unit.

“Vertical control” marks a precise elevation, rather than latitude/longitude. Hobbyist hunters often refer to all types of markers as “benchmarks,” although technically that is not correct. Only certain kinds of elevation marks are referred to as benchmarks (or bench marks) by surveyors.

This page shows some of the different types of markers: [https://thesurveystation.com/manual-of-geodetic-triangulation-excerpts/standard-marks/](https://thesurveystation.com/manual-of-geodetic-triangulation-excerpts/standard-marks/)

As you can probably tell by looking through my gallery of images (for example, 2014: [https://thesurveystation.com/year/2014](https://thesurveystation.com/year/2014)), I enjoy searching for survey markers of all kinds. If you are interested in looking for more, you might want to check out the Geocaching.com benchmark section:

[http://www.geocaching.com/mark/](http://www.geocaching.com/mark/)

Also, the National Geodetic Survey has an interactive map here:

[https://geodesy.noaa.gov/datasheets/ngs_map/](https://geodesy.noaa.gov/datasheets/ngs_map/)

If you can send a photo or some more information about any stamping on the mark, I can tell you more about it!

I hope this helps, but please don’t hesitate to ask more questions, and send an image of the mark (or its general location) if you can.
